content/review.rst

```diff
@@ -5,6 +5,9 @@ Review
 
 This page will discuss the code/changes review process as both the person
 who requests a review and a person who checks the review.
+Please note that the same review concepts/procedures apply to all platforms
+(Github, Trac, email, etc) and not only to code changes, but to any task
+you are working on.
```

Comment:
I think the last bit is redundant. What do you think about:
Please note that the process discussed here applies to all tickets and platforms (GitHub, Trac, email, etc), not only to code review.